jax/_src/scipy/linalg.py

```diff
@@ -1137,6 +1137,46 @@ def expm(A: ArrayLike, *, upper_triangular: bool = False, max_squarings: int = 1
 
   See Also:
     :func:`jax.scipy.linalg.expm_frechet`
+
+  Example:
+
+    If the matrices ``A`` and ``B`` satisfies ``A @ B = B @ A``, then
+    ``expm(A+B) = expm(A) @ expm(B)``
+
+    >>> A = jnp.array([[2, 0],
+    ...                [0, 1]])
+    >>> B = jnp.array([[3, 0],
+    ...                [0, 4]])
+    >>> with jnp.printoptions(precision=1, suppress=True):
+    ...     print(jax.scipy.linalg.expm(A+B))
+    [[148.4   0. ]
+     [  0.  148.4]]
+    >>> with jnp.printoptions(precision=1, suppress=True):
+    ...     print(jax.scipy.linalg.expm(A) @ jax.scipy.linalg.expm(B))
+    [[148.4   0. ]
+     [  0.  148.4]]
```

Comment:
You could even avoid printing these and just do the allclose

Reply:
Done